templates/repo/issue/filter_list.tmpl

```diff
@@ -1,3 +1,4 @@
+{{$queryLink := QueryBuild "?" "q" $.Keyword "type" $.ViewType "sort" $.SortType "state" $.State "labels" $.SelectLabels "milestone" $.MilestoneID "project" $.ProjectID "assignee" $.AssigneeID "poster" $.PosterID "archived" (Iif $.ShowArchivedLabels NIL)}}
```

Comment:
Would rename `QueryBuild` to `BuildQuery`, sounds more natural.

Reply:
The name is designed intentionally to match the "Query" family.

![image](https://github.com/user-attachments/assets/fdb055a5-2f32-4623-9f8f-c83766ba1111)

Reply:
But QueryEscape is the name of a golang stdlib function, while QueryBuild is not.

Reply:
They are the same family. And we use our own "queryEscape" function.

And all names I designed in Gitea are all clear enough, and most legacy rewriting and framework-level designs  are done by me, none of them causes any problem.

Reply:
It's okay I guess.